### ISSUE TITLE
deprecated functions in favor of streams API require better examples

### ISSUE THREAD

**Issue by scarf005** [denoland/std#3914]
**Is your feature request related to a problem? Please describe.**

https://github.com/denoland/deno_std/blob/9b9be2a25a8ae7cb4ce3835b1c7a4a4170163e22/streams/read_all.ts#L31-L34

It is not clear for users how the old code should be migrated to use web streams API.

**Describe the solution you'd like**

Add detailed migration guide in form of code block

**Describe alternatives you've considered**

**Comment by kt3k** [comment]
We probably need to create a comprehensive guide for migrating from Reader/Writer I/O to Web Stream based I/O, which includes the list of replacements of typical I/O operations such FS, Stdio, ChildProcess, etc

**Comment by iuioiua** [comment]
Perhaps, we could use https://github.com/denoland/denobyexample.

**Comment by muddlebee** [comment]
looks interesting. @iuioiua from my understanding we need to create some examples for below with code for both Reader/Writer I/O to Web Stream based I/O and how to migrate here https://github.com/denoland/denobyexample ?

> comprehensive guide for migrating from Reader/Writer I/O to Web Stream based I/O, which includes the list of replacements of typical I/O operations such FS, Stdio, ChildProcess, etc.

**Comment by iuioiua** [comment]
> looks interesting. @iuioiua from my understanding we need to create some examples for below with code for both Reader/Writer I/O to Web Stream based I/O and how to migrate here https://github.com/denoland/denobyexample ?
> 
> > comprehensive guide for migrating from Reader/Writer I/O to Web Stream based I/O, which includes the list of replacements of typical I/O operations such FS, Stdio, ChildProcess, etc.

Yes, that's correct. I'd be happy to guide someone through doing this. It's becoming more important that we do this as we approach the Deno Standard Library hitting stability.

**Comment by WasixXD** [comment]
@iuioiua Is this still needed? I'd be glad to help

**Comment by iuioiua** [comment]
There doesn't seem to be much demand for this now, from what I gather. WDYT, @kt3k?

**Comment by kt3k** [comment]
I agree. I see this kind of questions less frequently these days. Let's close
